homeassistant/components/openhardwaremonitor/sensor.py

```diff
@@ -75,6 +76,11 @@ def name(self):
         """Return the name of the device."""
         return self._name
 
+    @property
```

Comment:
Please use a static class attribute instead:

```python
class OpenHardwareMonitorDevice(SensorEntity):
    """Device used to display information from OpenHardwareMonitor."""

    _attr_state_class = SensorStateClass.MEASUREMENT

    def __init__(self, data, name, path, unit_of_measurement):
        """Initialize an OpenHardwareMonitor sensor."""
        ...
```

Reply:
Meh, I just copied the way it is implemented on another sensor, assuming that that's the way to do it. Guess I picked a bad example